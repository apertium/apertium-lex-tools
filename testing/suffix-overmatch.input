^prefixsuffix<cnjsub>/whocares<cnjsub>$ ^targetlm<adv>/rightlm<adv>/wronglm<adv>$
