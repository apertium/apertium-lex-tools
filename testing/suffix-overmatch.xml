<lrx>
  <rules>

    <rule>
      <match lemma="targetlm" tags="adv"><select lemma="rightlm"/></match>
    </rule>

    <rule weight="1.41421">
      <or>
        <match tags="sent.clb"/>
        <match lemma="suffix" tags="cnjsub"/>
      </or>
      <match lemma="targetlm" tags="adv"><select lemma="wronglm"/></match>
    </rule>

  </rules>
</lrx>
