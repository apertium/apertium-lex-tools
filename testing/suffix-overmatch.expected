^prefixsuffix<cnjsub>/whocares<cnjsub>$ ^targetlm<adv>/rightlm<adv>$
